### PR TITLE
Feature 21 client details card

### PR DESCRIPTION
**Pull request by DexterFreeman**
Created two components: BookingDetailsCard, ContactDetailsCard.
Tested both containers, also included mock data for both components in the test files. 
Styled according to Figma design. 
:shipit:

### CODE REVIEW

File: src/components/ContactDetailsCard/ContactDetailsCard.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./ContactDetailsCard.scss";
+
+const ContactDetailsCard = (props) => {
+  const {cardObject} = props
```
Comment (TristanPere) [inline src/components/ContactDetailsCard/ContactDetailsCard.jsx:5]: Put Props inline

File: src/components/BookingDetailsCard/BookingDetailsCard.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./BookingDetailsCard.scss";
+
+const BookingDetailsCard = (props) => {
+   const {cardObject} = props;
```
Comment (TristanPere) [inline src/components/BookingDetailsCard/BookingDetailsCard.jsx:5]: Destructure props inline, move _line 5_ into _line 4_

File: src/components/BookingDetailsCard/BookingDetailsCard.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+.booking-details-card * {
+    padding: 0;
+    margin: 0;
+}
+
+.booking-details-card{
+    width: 294px;
+    height: 307px;
+    background-color: #FFFFFF;
```
Comment (TristanPere) [inline src/components/BookingDetailsCard/BookingDetailsCard.scss:12]: Round Height and Width to nearest 5px

Comment (TristanPere) [inline src/components/BookingDetailsCard/BookingDetailsCard.scss:12]: Use the variables for colouring

File: src/components/BookingDetailsCard/BookingDetailsCard.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    display: flex;
+    flex-direction: column;
+    
+    gap: 5px;
+    margin-left: 100px;
+
+    &__heading{
+        font-size: 12px;
+        font-weight: 600;
+        color: #102B3F;
```
Comment (TristanPere) [inline src/components/BookingDetailsCard/BookingDetailsCard.scss:25]: use variables colouring

File: src/components/BookingDetailsCard/BookingDetailsCard.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        margin: 26px 0 44px 25px; 
+    }
+
+    &__subheading{
+        font-size: 10px;
+        font-weight: 500;
+        color: #102B3F;
```
Comment (TristanPere) [inline src/components/BookingDetailsCard/BookingDetailsCard.scss:32]: use variable for colouring

File: src/components/BookingDetailsCard/BookingDetailsCard.scss [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        margin-left: 25px
+    }
+
+    &__data{
+        font-size: 10px;
+        font-weight: 400;
+        color: #95ACCB;
```
Comment (TristanPere) [inline src/components/BookingDetailsCard/BookingDetailsCard.scss:39]: use variables for colouring

File: src/components/ContactDetailsCard/ContactDetailsCard.scss [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+.contact-card * {
+    padding: 0;
+    margin: 0;
+}
+
+.contact-card{
+    width: 294px;
+    height: 307px;
+    background-color: #FFFFFF;
```
Comment (TristanPere) [inline src/components/ContactDetailsCard/ContactDetailsCard.scss:12]: Same notes as before

**Review by TristanPere — CHANGES_REQUESTED** [review]
Looking strong. More nit-picks this time with sizing and colours. also there is a components folder in home that you made. can that be deleted please. Also please include the testing data you used in the next PR, if you used data from mockData.js please say.

File: src/components/BookingDetailsCard/BookingDetailsCard.scss [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+.booking-details-card * {
+    padding: 0;
+    margin: 0;
+}
+
+.booking-details-card{
+    width: 295px;
+    height: 310px;
+    background-color: $color-black;
```
Comment (TristanPere) [inline src/components/BookingDetailsCard/BookingDetailsCard.scss:12]: Needs to be white not black

**Review by TristanPere — CHANGES_REQUESTED** [review]

**Review by TristanPere — APPROVED** [review]

**Review by olirob93 — APPROVED** [review]

**Pull request merged**